mh/pat.py

```diff
@@ -2413,14 +2441,27 @@ def handle(self):
         self.server.add_to_debug(self)
         self.session = Session(self)
 
+        time.sleep(2)
```

Comment:
I know it’s not your fault but could you add back the comment saying that it’s a workaround for a dolphin bug